components/licence-title.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export function LicenceTitle ({
+	displayName = null,
+	isTrial = false,
+	isB2cPartnershipLicence = false,
+}){
+	if(isB2cPartnershipLicence || isTrial) {
+		return (
+			<h1 className="ncf__header ncf__center">
+				{
+					(displayName && (`${displayName}`))
+				}
+				{
+					isB2cPartnershipLicence
+						? 'Welcome to the Financial Times'
+						: (isTrial && ('Start your free trial'))
```

Comment:
redundant test, we already have `if(isB2cPartnershipLicence || isTrial) {` on line 9
```suggestion
						: 'Start your free trial'
```